packages/aws-cdk-lib/aws-rds/lib/aurora-cluster-instance.ts

```diff
@@ -456,7 +456,7 @@ class AuroraClusterInstance extends Resource implements IAuroraClusterInstance {
 
     const instanceParameterGroup = props.parameterGroup ?? (
       props.parameters
-        ? new ParameterGroup(props.cluster, 'InstanceParameterGroup', {
+        ? new ParameterGroup(props.cluster, `${id}InstanceParameterGroup`, {
```

Comment:
This code change will replace the logical name of a ParameterGroup that has already been created by specifying `props.parameters` so far.

If we want to prevent this, I think there is another way.

- Check if 'InstanceParameterGroup' has already been created in the stack.
  - e.g. `if (Stack.of(this).node.tryFindChild('InstanceParameterGroup') === undefined)`
- Create 'InstanceParameterGroup' only if it is.
- If not, create an `${id}InstanceParameterGroup`.

However, this is not intuitive, and unexpected resource substitutions will occur if the order in the `readers` array is changed. Also, if we create an 'InstanceParameterGroup' ParameterGroup elsewhere, there will be a conflict.

For these reasons, I stopped.